src/lib/merkle_ledger/converting_merkle_tree.ml

```diff
@@ -169,6 +169,25 @@ end = struct
 
   let get_directory t = Primary_ledger.get_directory t.primary_ledger
 
+  let make_checkpoint t ~directory_name =
+    (* Must checkpoint both databases for recovery *)
+    let converting_dir = directory_name ^ "_converting" in
```

Comment:
This is not particularly good to have duplicated - this is done again in `create_checkpoint` below, and also in https://github.com/MinaProtocol/mina/pull/17428.